lesson_29/script.js

```diff
@@ -0,0 +1,10 @@
+let isImgZoommed = document.querySelector('.gallery');
```

Comment:
`isImgZoommed` variable sounds like something which has to contain `boolean`, but not html element

so rename you variable to `galleryContainer` for example